src/content/events/vision-weekend-usa-2024.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Vision Weekend"
```

Comment:
Hi @oliviasigler 

Should the title be `Vision Weekend 2024` or `Vision Weekend USA 2024`, to be consistent with the previous editions?

Reply:
I adapted it and merged.